packages/tailwind/docs/MediaQueries.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react"
+import { Story } from "@storybook/react"
+import { withDesign } from "storybook-addon-designs"
+import { Divider } from "@kaizen/draft-divider"
+import { Heading, Paragraph } from "@kaizen/typography"
+import { figmaEmbed } from "../../../storybook/helpers"
+
+export default {
+  title: "Tailwind/Psuedo Selectors/MediaQueries",
```

Comment:
Was the intention here to have the media queries story nested in a pseudo selectors folder? If so should the hover and focus state story be in here too?

Also, you can 'hoist' up the story to reduce how deep it's nested by matching the title with storyname, ie:
```
title: "Tailwind/Psuedo Selectors/Media Queries",

... ln 84
TailwindMediaQueries.storyName = "Media Queries"
```

This just means you won't have a single story within a folder within a folder.

Reply:
<img width="236" alt="Screenshot 2023-01-11 at 10 54 17 am" src="https://user-images.githubusercontent.com/36558508/211687760-f49788de-95a2-4b26-a57f-40c8fdc1d14b.png">

Reply:
Yeah good pickup - I was originally planning on moving the other selectors into that folder, but realised that our `hover`/ `focus` stories aren't demonstrating anything related to our package (they just show a TW feature). So I'm wondering if we should remove those ones and just have media queries at the top level.

Media queries is a little different I think, because we've set our own breakpoints. So worth keeping these ones.